docs/content/nuget-dependencies.md

```diff
@@ -36,14 +36,14 @@ source http://myserver/nuget/api/v2 // Custom feed.
 It's also possible to provide login information for private NuGet feeds:
 
 ```paket
-source http://example.com/nuget/api/v2 username: "user name" password: "the password"
+source http://example.com/nuget/api/v2 username: "user name" password: "the password" authtype: "basic"
 ```
 
 If you don't want to check your username and password into source control, you
 can use environment variables instead:
 
 ```paket
-source http://myserver/nuget/api/v2 username: "%PRIVATE_FEED_USER%" password: "%PRIVATE_FEED_PASS%"
+source http://myserver/nuget/api/v2 username: "%PRIVATE_FEED_USER%" password: "%PRIVATE_FEED_PASS%" authtype: "ntlm"
```

Comment:
No, the default credentials aren't used - just the authentication scheme.  See my reply to your other comment. :)